### PR TITLE
EWPP-916: Drupal 9 compatibility fixes

### PR DESCRIPTION
**Pull request by yenyasinn**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,33 +6,32 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.8",
+        "drupal/core": "^8.9",
         "ext-json": "*",
-        "php": ">=7.2"
+        "php": ">=7.3"
```
Comment (yenyasinn) [inline composer.json:11]: It will break tests for all environments where PHP 7.2 is used. But officially Drupal 8 and PHP 7.2 aren't supported anymore.

**Comment by yenyasinn** [comment]
Work is suspended because of dependencies from oe_media and oe_multilingual.

File: .drone.yml [hunk 2]
```diff
@@ -94,5 +105,5 @@ matrix:
     - lowest
     - highest
   PHP_VERSION:
-    - 7.2
     - 7.3
+    - 7.4
```
Comment (yenyasinn) [inline .drone.yml:109]: We don't support 7.4 for now.

File: .drone.yml [hunk 3]
```diff
@@ -82,6 +82,17 @@ pipeline:
       - ./vendor/bin/behat --strict
       - ./vendor/bin/behat --strict --profile=static
 
+  test-drupal9-compatibility:
```
Comment (yenyasinn) [inline .drone.yml:85]: This step is redundant

File: composer.json (hunk 1 above)
Comment (yenyasinn) [reply composer.json:11]: We don't support PHP 7.2 anymore.

File: modules/oe_webtools_media/tests/src/FunctionalJavascript/MediaSourceWebtoolsTest.php [hunk 4]
```diff
@@ -32,52 +32,56 @@ class MediaSourceWebtoolsTest extends MediaSourceTestBase {
...
+   * {@inheritdoc}
    */
-  public function testMediaWebtoolsSource(string $widget_type, string $widget_name, string $service, string $thumbnail_filename): void {
-    $media_type_id = 'test_media_webtools_type';
+  protected $defaultTheme = 'classy';
 
+  /**
+   * Tests the webtools media source.
+   */
+  public function testMediaWebtoolsSource(): void {
```
Comment (upchuk) [inline modules/oe_webtools_media/tests/src/FunctionalJavascript/MediaSourceWebtoolsTest.php:42]: This is a test refactor as we were using a data provider in a JS functional test which would take forever. So I looped over the data set.

**Review by yenyasinn — COMMENTED** [review]
I can't approve my PR. It looks good for me.

**Review by drishu — APPROVED** [review]

**Review by sergepavle — APPROVED** [review]

**Review by upchuk — APPROVED** [review]

**Pull request merged**